recipes/pypairs/meta.yaml

```diff
@@ -1,36 +1,60 @@
 {% set name = "pypairs" %}
-{% set version = "2.0.6" %}
+{% set version = "3.0.9" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 39546c372405f22cefd2b00fdbf0bb2bd6abe779d88a35f82f4b746807446bcb
+  sha256: baf268424ebe0eb58339e41877531c182e4c6634f59e9c21b77e0f33ec2f634d
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
-  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py2k]
 
 requirements:
   host:
+    - anndata >=0.6.13
+    - colorama
+    - h5py >=2.8.0
+    - numba >=0.40.1
+    - numpy >=1.15.4
+    - pandas >=0.23.4
     - pip
+    - psutil
     - python
+    - scikit-learn
   run:
+    - anndata >=0.6.13
     - colorama
-    - numba
-    - numpy
-    - pandas
+    - h5py >=2.8.0
+    - numba >=0.40.1
+    - numpy >=1.15.4
+    - pandas >=0.23.4
+    - psutil
     - python
+    - scikit-learn
 
 test:
   imports:
     - pypairs
+    - pypairs.datasets
+    - pypairs.plotting
+    - pypairs.preprocessing
+    - pypairs.tests
+    - pypairs.tools
 
 about:
   home: https://github.com/rfechtner/pypairs
-  license: MIT
-  license_family: MIT
-  summary: A Python-reimplementation of the Pairs algorithm described by A. Scialdone et al. (2015)
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: A python scRNA-Seq classifier
+  doc_url: https://pypairs.readthedocs.io/
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - 11939699
```

Comment:
What is this? You can add your github user name if you like.

Reply:
Ok... I got this from running `conda skeleton pypi pypairs`. There the placeholder stated to fill in the user ID